packages/admin/cms-admin/src/generator/future/generateForm/generateFormField.ts

```diff
@@ -148,7 +148,7 @@ export function generateFormField({
     let startAdornment: AdornmentData = { adornmentString: "" };
     let endAdornment: AdornmentData = { adornmentString: "" };
 
-    if (config.startAdornment) {
+    if ("startAdornment" in config && config.startAdornment) {
```

Comment:
I believe the two conditions in this if do the same 🤔

Reply:
No:
- `"startAdornment" in config` checks if the key `startAdornment` exists in the `config` object
- `config.startAdornment` is to check, if it has a non-falsy value

The first part is required to allow the second part to work. 
This is needed because `config` is typed as a union of types, where some of them define `startAdornment` and some do not.

Reply:
TLDR: `"startAdornment" in config` narrows the type of the `config` object.